assets/js/blocks/product-category/block.js

```diff
@@ -37,8 +37,13 @@ class ProductByCategoryBlock extends Component {
 		this.state = {
 			products: [],
 			loaded: false,
+			changedAttributes: null,
+			isEditing: false,
```

Comment:
I was looking at `ReusableBlockEdit` for inspiration and from past work I've seen this type of thing as state. I'm not sure why we wouldn't use state for editMode/isEditing - is there a reason I'm missing?

Reply:
That's a good question. @ryelle do you know if there are any pros/cons between using attributes or state for this?